Course_Selection_Main/forms.html

```diff
@@ -4,123 +4,67 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/resources/css/base.css">
-    <link rel="stylesheet" href="/resources/css/forms.css">
-    <title>Forms - Elgin Park Secondary</title>
-    <link rel="canonical" href="https://www.elginparksecondary.com/forms">
-    <link rel="icon" type="image/png" href="/fav/favicon-96x96.png" sizes="96x96">
-    <link rel="icon" type="image/svg+xml" href="/fav/favicon.svg">
-    <link rel="shortcut icon" href="/fav/favicon.ico">
-    <link rel="apple-touch-icon" sizes="180x180" href="/fav/apple-touch-icon.png">
-    <meta name="apple-mobile-web-app-title" content="Elgin Park">
-    <link rel="manifest" href="/fav/site.webmanifest">
+    <link rel="stylesheet" href="top_content.css">
+    <link rel="stylesheet" href="forms.css">
+    <link rel="icon" type="image/x-icon" href="elgin_logo.png">
+    <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
+    <meta http-equiv="Pragma" content="no-cache" />
+    <meta http-equiv="Expires" content="0" />
+    <title>Elgin Park Secondary</title>

```

Comment:
```suggestion
    <link rel="stylesheet" href="/resources/css/base.css">
    <link rel="stylesheet" href="/resources/css/forms.css">
    <title>Forms - Elgin Park Secondary</title>
    <link rel="canonical" href="https://www.elginparksecondary.com/forms">
    <link rel="icon" type="image/png" href="/fav/favicon-96x96.png" sizes="96x96">
    <link rel="icon" type="image/svg+xml" href="/fav/favicon.svg">
    <link rel="shortcut icon" href="/fav/favicon.ico">
    <link rel="apple-touch-icon" sizes="180x180" href="/fav/apple-touch-icon.png">
    <meta name="apple-mobile-web-app-title" content="Elgin Park">
    <link rel="manifest" href="/fav/site.webmanifest">
```
This should not be deleted, also it uses old paths